docset/winserver2012r2-ps/adcsdeployment/install-adcsnetworkdeviceenrollmentservice.md

```diff
@@ -45,47 +45,37 @@ Int is equivalent to Int32 in the .NET Frameworkhttp://msdn.microsoft.com/en-us/
 
 ## EXAMPLES
 
-### -------------------------- EXAMPLE 1 --------------------------
+### Example 1: Display the default NDES settings
 ```
-C:\PS>Install-AdcsNetworkDeviceEnrollmentService -ApplicationPoolIdentity -WhatIf
+PS C:\> Install-AdcsNetworkDeviceEnrollmentService -ApplicationPoolIdentity -WhatIf
 ```
 
-Description
-
------------
-
-This command displays the default Network Device Enrollment Service settings when the service is running as the default application identity without making any changes to the configuration.
+This command displays the default NDES settings that will be configured if it is installed.
 
-### -------------------------- EXAMPLE 2 --------------------------
+### Example 2: Display the default NDES settings using a service account name and password
 ```
-C:\PS>Install-AdcsNetworkDeviceEnrollmentService -ServiceAccountName <Domain>\<AccountName> -ServiceAccountPassword (read-host "Set user password" -assecurestring) -WhatIf
+PS C:\> Install-AdcsNetworkDeviceEnrollmentService -ServiceAccountName "CONTOSO\PattiFul" -ServiceAccountPassword (read-host "Set user password" -assecurestring) -WhatIf
```

Comment:
Could you change the account name to something like "CONTOSO\svcNDES" so it is clear that it is a service account rather than a potential user account? Same applies to the other example/description.